Update README.md example code to reflect fix for issue 25

pmotyka opened the pull request.

timneedham commented:
Thanks all... the README could do with a bit of TLC..

Isn't it:

`npm install @wmfs/statebox` --save now?

`statebox.waitUntilStoppedRunning()` is superseded with 

```
const executionDescription = await statebox.startExecution(
  {
    numbers: {
      number1: 3,
      number2: 2
    },
    operator: '-'
  }, // input
  'calculator', // state machine name
  {
    sendResponse: 'COMPLETE'
  } // options
)
```

And a section on the `options` wouldn't hurt? 😀

pmotyka commented:
Good suggestions, @timneedham.  I've updated to the npm command and removed the unnecessary  statebox.waitUntilStoppedRunning(...).  Re: options, I'm not seeing much that is governed by them, other than a hasDelayedResponse check.  I'll defer to someone with more expertise on those incase I'm overlooking additional functionality there.
